spec/services/auctions_closed_within_last_twenty_four_hours_finder_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe AuctionsClosedWithinLastTwentyFourHoursFinder do
+  describe '#perform' do
+    it 'returns auctions with an end datetime that occured today' do
+      Timecop.freeze(Time.parse("19:00:00 UTC")) do
+        _old_auction = create(:auction, ended_at: 2.days.ago)
+        newly_closed_auction = create(:auction, ended_at: 2.hours.ago)
+        _future_auction = create(:auction, ended_at: 2.days.from_now)
```

Comment:
sure -- btw the reason I suggested doing a tighter timeframe is that we've dealt with some crazy stuff related to clockwork / timezones on Dolores. Basically, we had an issue where we were looking for the "current date" in a different timezone so things were getting sent at midnight ET.....if we'd had a test that included records *just* outside of the timeframe that we wanted, the bug would have been avoided